### PR TITLE
changed feature

### PR DESCRIPTION
**Pull request by AleksandraZelik**
made feature

### CODE REVIEW

File: newFile.txt [hunk 1]
```diff
@@ -1,10 +1,4 @@
 Wow
 Somewefwefwefwefwef
 wecwecwecwecwecwecw
-ecwecwec
```
Comment (ZelikSV) [inline newFile.txt:4]: Why do we remove it?

**Review by ZelikSV — CHANGES_REQUESTED** [review]
Need to be fixed

File: newFile.txt (hunk 1 above)
Comment (AleksandraZelik) [reply newFile.txt:4]: i remove this

**Review by ZelikSV — APPROVED** [review]
All was fixed

**Review by ZelikSV — CHANGES_REQUESTED** [review]
Need to be fixed

File: newFile.txt [hunk 2]
```diff
@@ -1,10 +1,5 @@
...
-cwecwec
-wecwecwecw
-ecwec
-
-wecwecwecwe
-cwecwec
+1233546765878
+1235465677980
```
Comment (ZelikSV) [inline newFile.txt:5]: Replace pls to '999-00-888-444'

**Review by ZelikSV — APPROVED** [review]

**Pull request merged**